app/src/examples/IPodExample.tsx

```diff
@@ -181,6 +161,57 @@ export default function IPodExample() {
   );
 }
 
+type SongCoverProps = {
+  artist: string;
+  song: string;
+  index: number;
+  position: SharedValue<number>;
+  itemTotalSize: number;
+  borderMargin: number;
+};
+
+function SongCover({
+  artist,
+  song,
+  index,
+  position,
+  itemTotalSize,
+  borderMargin,
+}: SongCoverProps) {
+  const uas = useAnimatedStyle(() => {
```

Comment:
It's just ugly here like this 😅 
```suggestion
  const animatedStyle = useAnimatedStyle(() => {
```